x-pack/plugins/profiling/public/components/flame_graphs_view/get_impact_rows.ts

```diff
@@ -23,21 +24,26 @@ const CO2_PER_KWH = 0.92;
 const CORE_COST_PER_HOUR = 0.0425;
 
 export function getImpactRows({
+  countInclusive,
+  countExclusive,
   samples,
-  childSamples,
-  sampledTraces,
-  totalTraces,
+  totalSamples,
   totalSeconds,
 }: {
+  countInclusive: number;
+  countExclusive: number;
   samples: number;
-  childSamples: number;
-  sampledTraces: number;
-  totalTraces: number;
+  totalSamples: number;
   totalSeconds: number;
 }) {
-  const percentage = samples / sampledTraces;
-  const percentageNoChildren = (samples - childSamples) / sampledTraces;
-  const totalCoreSeconds = totalTraces / 20;
+  const samplingRate = samples / totalSamples;
+
+  const upscaledCountInclusive = countInclusive / samplingRate;
+  const upscaledCountExclusive = countExclusive / samplingRate;
+
+  const percentage = upscaledCountInclusive / totalSamples;
+  const percentageNoChildren = upscaledCountExclusive / samplingRate / totalSamples;
+  const totalCoreSeconds = totalSamples / 20;
```

Comment:
Is this the sampling rate of the host agent (in Hz)?

Could you replace this with a constant and an appropriate comment?

Reply:
It is the sampling rate in Hz, but in other words it also could be a "coreSecond".
See my comment below.